client/web/src/site-admin/UserManagement/components/DropdownPagination.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useCallback } from 'react'
+
+import { mdiChevronRight, mdiChevronLeft } from '@mdi/js'
+import classNames from 'classnames'
+
+import { Button, Text, Icon, Popover, PopoverContent, PopoverOpenEvent, PopoverTrigger } from '@sourcegraph/wildcard'
+
+export interface DropdownPaginationProps {
+    /**
+     * The total number of items.
+     */
+    total: number
+    /**
+     * The current offset.
+     */
+    offset: number
+    /**
+     * The current limit.
+     */
+    limit: number
+    /**
+     * The limit options to display in the dropdown.
+     */
+    options?: ({ limit: number; label: string } | number)[]
+    /**
+     * Callback for when the offset changes.
+     */
+    onOffsetChange: (offset: number) => void
+    /**
+     * Callback for when the limit changes.
+     */
+    onLimitChange: (limit: number) => void
+    /**
+     * A function for formatting the label to display.
+     */
+    formatLabel?: (start: number, end: number, total: number) => string
+    className?: string
+}
+
+/**
+ * A dropdown pagination component that allows users to change the limit and offset of a paginated list.
+ */
+export const DropdownPagination: React.FunctionComponent<DropdownPaginationProps> = ({
+    limit,
+    offset,
+    total,
+    options,
+    onLimitChange,
+    onOffsetChange,
+    formatLabel = (start: number, end: number, total: number) => `Showing ${start}-${end} items of ${total}`,
+    className,
+}) => {
+    const [isOpen, setIsOpen] = useState<boolean>(false)
+    const handleOpenChange = useCallback((event: PopoverOpenEvent): void => {
+        setIsOpen(event.isOpen)
+    }, [])
+
+    const handlePreviousPage = useCallback(
+        () => onOffsetChange(Math.max(offset - limit, 0)),
+        [limit, offset, onOffsetChange]
+    )
+    const handleNextPage = useCallback(() => {
+        const newOffset = offset + limit
+        if (total > newOffset) {
+            onOffsetChange(newOffset)
+        }
+    }, [limit, offset, total, onOffsetChange])
+
+    return (
+        <div className={classNames('d-flex justify-content-between align-items-center', className)}>
+            <Button className="mr-2" onClick={handlePreviousPage}>
+                <Icon aria-label="Show previous icon" svgPath={mdiChevronLeft} />
```

Comment:
Since we do not have a label for the button, it might be enough to just say what the button does here instead of saying the `icon` word.
```suggestion
                <Icon aria-label="Show previous" svgPath={mdiChevronLeft} />
```